rotkehlchen/tests/api/test_tags.py

```diff
@@ -660,13 +654,7 @@ def test_delete_utilized_tag(rotkehlchen_api_server):
             "blockchainsaccountsresource",
             blockchain='BTC',
         ), json={'accounts': accounts_data})
-    result = assert_proper_response_with_result(response)
-    assert_btc_balances_result(
```

Comment:
Why did you delete this and where did this test go?

Reply:
If we add an extra blockchain balance query here, it would be pretty similar to, for example, this test https://github.com/rotki/rotki/blob/develop/rotkehlchen/tests/api/test_blockchain.py#L193